builder/vsphere/iso/step_remove_floppy.go

```diff
@@ -42,6 +42,7 @@ func (s *StepRemoveFloppy) Run(_ context.Context, state multistep.StateBag) mult
 			state.Put("error", err)
 			return multistep.ActionHalt
 		}
+		state.Remove("uploaded_floppy_path")
```

Comment:
Every time the build failed after this step, for example at export or import to the content library, when running the StepAddFloppy cleanup that would fail because the floppy was already removed by the StepRemoveFloppy. Here we make sure to remove the key from the state and avoid cleanup the already removed file.